frontend/src/views/internal/electrification/project/ProjectView.vue

```diff
@@ -415,37 +443,126 @@ onBeforeMount(async () => {
       <TabPanel :value="2">
         <div class="flex items-center pb-5">
           <div class="grow">
-            <p class="font-bold">{{ t('i.common.projectView.applicablePermits') }} ({{ getPermits.length }})</p>
+            <p class="font-bold">{{ t('i.common.projectView.applicableAuthorizations') }} ({{ getPermits.length }})</p>
           </div>
           <Button
             aria-label="Add permit"
             :disabled="isCompleted || !useAuthZStore().can(Initiative.ELECTRIFICATION, Resource.PERMIT, Action.CREATE)"
-            @click="permitModalVisible = true"
+            @click="
+              router.push({
+                name: RouteName.INT_ELECTRIFICATION_PROJECT_AUTHORIZATION,
+                query: {
+                  activityId: activityId,
+                  projectId: projectId
+                }
+              })
+            "
           >
             <font-awesome-icon
               class="pr-2"
               icon="fa-solid fa-plus"
             />
-            Add permit
+            {{ t('i.common.projectView.addAuthorization') }}
           </Button>
         </div>
+        <!-- On going Authorizations -->
         <div
-          v-for="(permit, index) in getPermits"
+          v-for="(permit, index) in getPermits.filter((p) =>
+            [PermitAuthorizationStatus.IN_REVIEW, PermitAuthorizationStatus.PENDING].includes(
+              p.authStatus as PermitAuthorizationStatus
+            )
+          )"
           :key="permit.permitId"
           :index="index"
-          class="mb-6"
+          class="mb-6 mt-6"
         >
-          <PermitCard
+          <AuthorizationCard
             :editable="!isCompleted"
             :permit="permit"
+            @authorization-card:more="
+              router.push({
+                name: RouteName.INT_HOUSING_PROJECT_AUTHORIZATION,
+                query: {
```

Comment:
Why the query over params? We should be utilizing the pattern we've established for other routes, ids should be params, activity Id can be retrieve from the project or permit store.

Reply:
I'm still seeing the query instead of params in the code?